$MirOS: ports/sysutils/e2fsprogs/patches/patch-lib_blkid_devno_c,v 1.1.7.1 2005/03/18 15:52:04 tg Exp $
--- lib/blkid/devno.c.orig	Sat May  7 18:34:31 2005
+++ lib/blkid/devno.c	Tue Jan  3 02:52:57 2006
@@ -224,7 +224,7 @@ int main(int argc, char** argv)
 		}
 		devno = makedev(major, minor);
 	}
-	printf("Looking for device 0x%04Lx\n", devno);
+	printf("Looking for device 0x%04Lx\n", (unsigned long long)devno);
 	devname = blkid_devno_to_devname(devno);
 	if (devname)
 		free(devname);
